app/resources/v1/ont/instrument_resource.rb

```diff
@@ -2,10 +2,26 @@
 
 module V1
   module Ont
-    # InstrumentResource
+    # @todo This documentation does not yet include a detailed description of what this resource represents.
+    # @todo This documentation does not yet include detailed descriptions for relationships, attributes and filters.
+    # @todo This documentation does not yet include any example usage of the API via cURL or similar.
+    #
+    # @note Access this resource via the `/api/v1/ont/instrument/` endpoint.
```

Comment:
Again don't think this is a valid endpoint, but `https://<traction>/v1/ont/instruments/` or `https://<traction>/v1/ont/instruments/<instrument_id>` is etc